service/src/test/java/bio/terra/workspace/service/workspace/WorkspaceServiceTest.java

```diff
@@ -217,7 +262,7 @@ void duplicateWorkspaceUserFacingIdRequestsRejected() {
             DuplicateUserFacingIdException.class,
             () -> workspaceService.createWorkspace(duplicateUserFacingId, USER_REQUEST));
     assertEquals(
-        String.format("Workspace with user facing ID %s already exists", userFacingId),
+        String.format("Workspace with ID %s already exists", userFacingId),
```

Comment:
In the logs, I think it's important to distinguish these, since they won't always look different.